### DISCUSSION TITLE
How are you using the Open Grid Emissions Initiative?

### DISCUSSION THREAD

**Discussion by grgmiller** [singularity-energy/open-grid-emissions#203] [General]
We would love to hear about how you plan to use (or are currently using) the Open Grid Emissions Initiative! We've identified what we believe some potential use cases in the below graphic, but hearing from you helps us better understand how we can make the data most relevant and prioritize our efforts for the next release. 

![Use Case Table](https://user-images.githubusercontent.com/45949268/187002649-265f3f15-d9e9-4093-b12d-65a3c0d53763.PNG)

**Comment by eforkin** [comment]
Hi @grgmiller, I'm a software engineer at Indeed and part of our Green Engineering Guild, a working group focused on sustainability in software development. We're looking for ways to ultimately plug in a date range to get the carbon intensity of regions our products are in use at an hourly level. To do that ourselves, we want to create an open-source [Impact Framework](https://if.greensoftware.foundation/) plugin that will allow for real-time dates as well (ex: hourly intensity yesterday). We know the EIA provides real-time data like that, but the open grid emissions initiative collects data on a yearly basis most recently having 2022 data. If we were to attempt a fork of the project, is there anything we'd lose from your methodology if, to use the same example, we wanted hourly data for yesterday?
